.github/dependabot.yml

```
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Enable version updates for Python dependencies
+  - package-ecosystem: "pip"
```

This uses uv, not pip directly.

```suggestion
  - package-ecosystem: "uv"
```